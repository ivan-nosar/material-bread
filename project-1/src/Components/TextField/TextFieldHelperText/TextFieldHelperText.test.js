import React from 'react';
import { TextFieldHelperText } from '../../..';

import renderer from 'react-test-renderer';

test('TextFieldHelperText Renders', () => {
  const tree = renderer.create(<TextFieldHelperText />).toJSON();
  expect(tree).toMatchSnapshot();
});
