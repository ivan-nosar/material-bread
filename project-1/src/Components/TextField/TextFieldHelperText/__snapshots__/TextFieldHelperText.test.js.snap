// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFieldHelperText Renders 1`] = `null`;
